### PR TITLE
ROC-7170 - validate optional contact name field to not have email address

### PR DESCRIPTION
**Pull request by akritis22**
### JIRA link
https://tools.hmcts.net/jira/browse/ROC-7170

### Change description
when I select an organisation or company as the defendant or claimant type and I attempt to enter in an email address into the contact name field it will display an error message.

Added isNotEmail validator in cmc-validators to support this change

### Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: src/main/app/forms/models/companyDetails.ts [hunk 1]
```diff
@@ -1,9 +1,11 @@
 import { PartyDetails } from './partyDetails'
 import { PartyType } from 'common/partyType'
 import { MaxLength } from '@hmcts/cmc-validators'
+import { IsNotEmail } from '@hmcts/cmc-validators/dist/isNotEmail'
```
Comment (andrewwa-kainos) [inline src/main/app/forms/models/companyDetails.ts]: Shouldn't need `/dist/isNotEmail` - merge in with the `MaxLength` import

File: src/main/app/forms/models/organisationDetails.ts [hunk 2]
```diff
@@ -1,9 +1,11 @@
 import { PartyDetails } from './partyDetails'
 import { PartyType } from 'common/partyType'
 import { MaxLength } from '@hmcts/cmc-validators'
+import { IsNotEmail } from '@hmcts/cmc-validators/dist/isNotEmail'
```
Comment (andrewwa-kainos) [inline src/main/app/forms/models/organisationDetails.ts]: As above, shouldn't need `/dist/isNotEmail` - merge in with the `MaxLength` import

File: src/test/app/forms/models/organisationDetails.ts [hunk 3]
```diff
@@ -94,6 +94,13 @@ describe('OrganisationDetails', () => {
       expectValidationError(errors, OrganisationDetailsValidationErrors.CONTACT_PERSON_NAME_TOO_LONG.replace('$constraint1', '30'))
     })
 
+    it('should return error when contact person has email address', () => {
+      organisationDetails.contactPerson = 'test@domian.com'
```
Comment (andrewwa-kainos) [inline src/test/app/forms/models/organisationDetails.ts]: Typo, domain

File: src/main/app/forms/models/organisationDetails.ts (hunk 2 above)
Comment (akritis22) [reply src/main/app/forms/models/organisationDetails.ts]: When I do that it throws compilation error that IsnotEmail not found in @hmcts/cmc-validators. Am I missing something?

Comment (andrewwa-kainos) [reply src/main/app/forms/models/organisationDetails.ts]: Update cmc-validators to 0.2.6. Apologies for my publishing mistake!

**Comment by daveoh** [comment]
@akritis22 Can you resolve the conflicts?

**Pull request closed without merging**